src/component/mxgraph/ShapeConfigurator.ts

```diff
@@ -52,7 +58,6 @@ export default class ShapeConfigurator {
     this.mxShape.prototype.createSvgCanvas = function() {
       // TODO should be 'typeof mxgraph.mxSvgCanvas2D', current type definition does not declare 'minStrokeWidth'
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
:warning: this comment is not accurate anymore, as the type is now managed by the declared const above (see comment below as well)